src/views/Login/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
```

Comment:
index.js should just import and export. I recommend moving all of this code to a Login.js file, and then just importing and exporting that in this file. For an example, see the files in Layout 🎉